Platform.XamarinIOS/Realm.Sync.XamarinIOS/ApiDefinition.cs

```diff
@@ -0,0 +1,62 @@
+﻿// WARNING even though this file contains apparently generic content with no function
```

Comment:
It's the same as in `Realm.XamarinIOS`. Since it basically comes from the Xamarin Studio template, I don't know if we need or want to have the license header :/